sections/collection-list.liquid

```diff
@@ -68,10 +64,15 @@
         </div>
       {%- endif -%}
     </slider-component>
-  
+
     {%- if section.settings.show_view_all and section.blocks.size < collections.size -%}
-      <div class="center collection-list-view-all{% if show_mobile_slider %} small-hide medium-hide{% endif %}">
-        <a href="{{ routes.collections_url }}" class="button">{{ 'sections.collection_list.view_all' | t }}</a>
+      <div class="center">
+        <a href="{{ routes.collections_url }}"
+          class="view-all-button{% if section.settings.view_all_style == 'link' %} link underlined-link{% elsif section.settings.view_all_style == 'solid' %} button{% else %} button button--secondary{% endif %}"
```

Comment:
```suggestion
          class="view-all-button {% if section.settings.view_all_style == 'link' %}link underlined-link{% elsif section.settings.view_all_style == 'solid' %}button{% else %}button button--secondary{% endif %}"
```

Reply:
I kinda like the space within the checks to give it a bit more breathing room. Something we could talk about as a team when we discuss our coding standards. 👍 I will leave it as is for now.